feat: FIT-582: Members Page performance improvement to display only users with contributions

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/17171328859)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/536
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17151309178

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17151444984

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17157605397

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17161812066

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-fit-582'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17161812066

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17161919678

hakan458 commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17162314630)

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17162265734

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-fit-582'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17162265734

hakan458 committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17162314630

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17162736512

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17166106437

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-fit-582'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17166106437

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17166471438

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17166515220

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17166934479

hakan458 commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17168489485)

hakan458 committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17168489485

hakan458 committed:
Merge branch 'develop' into 'fb-fit-582'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17168489485

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17168655632

hakan458 commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17171282414)

hakan458 committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17171282414

hakan458 committed:
Merge branch 'develop' into 'fb-fit-582'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17171282414

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17171328859

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/17217255925
